crates/builder-api/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "hs-builder-api"
```

Comment:
For consistency I feel like we should probably call this `hotshot-builder-api` (I think everything else related to HotShot is called `hotshot-`). But I think we should do that separately, in another PR.

@move47 Thoughts?

Reply:
Yeah, I had this also. We have to change in several other crates too, so thought of doing altogether.

Reply:
Oh I added it to this pr, hopefully that's ok